.github/workflows/check-golangupx-release.yml

```diff
@@ -101,10 +101,19 @@ jobs:
           sed -i "s/go-version: '[0-9]\+\.[0-9]\+\.[0-9]'/go-version: '${NEW_VERSION}'/" .github/workflows/release_build.yml
           echo "Updated release_build.yml Go version ${NEW_VERSION}"
 
-      - name: Commit updated files
+      - name: Create Pull Request
         run: |
+          set -x
+          git branch golangupx_new_release
+          git switch golangupx_new_release
           git config --local user.email "github-actions[bot]@users.noreply.github.com"
           git config --local user.name "github-actions[bot]"
           git add Dockerfile go.mod .github/workflows/release_build.yml
-          git commit -m "chore: update Golang upx to version ${{ needs.check-golang-version.outputs.version }}" || echo "No changes to commit"
-          git push || echo "No changes to push"
+          git commit -m "chore: update Golang upx to version ${{ needs.check-golang-version.outputs.version }}"
+          git push --force -u origin golangupx_new_release
```

Comment:
I do not think the --force flag is required

Reply:
see my comment [here](https://github.com/devops-works/scan-exporter/pull/29#discussion_r2453047577); by checking early if the PR for the current version already exist, we can avoid pushing multiple times to an existing PR